digdag-docs/src/command_reference.rst

```diff
@@ -350,6 +350,8 @@ In the config file, following parameters are available
 * log-server.s3.credentials.secret-access-key (string. default: instance profile)
 * log-server.s3.path-style-access (boolean. default: false)
 * digdag.secret-encryption-key = (base64 encoded 128-bit AES encryption key)
+* executor.task_ttl (A task kills if a single task is running longer than this period. string. default: 1d)
```

Comment:
Could you change `kills` to `is killed`? 

Also, it might be better to put a type first considering other items above. How about this?
```
(string. default: 1d. A task is killed if it runs more than this period)
```

Reply:
@komamitsu Thank you for the reply. Fixed.